common/src/main/java/io/pravega/example/tensorflow/Main.java

```diff
@@ -0,0 +1,67 @@
+package io.pravega.example.tensorflow;
+
+
+import org.apache.commons.io.IOUtils;
+
+import javax.imageio.ImageIO;
+import java.awt.image.BufferedImage;
+import java.io.ByteArrayInputStream;
+import java.io.File;
+import java.io.FileInputStream;
+import java.io.IOException;
+import java.util.List;
+
+//import sun.nio.ch.IOUtil;
+
+public class Main {
```

Comment:
This appears to be test code. Move this to common/src/test/java/... and use org.junit. See tests in flinkprocessor.